src/js/actions/layers.js

```diff
@@ -1490,6 +1492,29 @@ define(function (require, exports) {
     revealLayers.writes = [locks.PS_DOC, locks.JS_DOC];
 
     /**
+     * Reveal and select the vector mask of the selected layer. 
+     * Also switch to the vector based superselect tool.
+     *
+     * @return {Promise}
+     */
+    var editVectorMask = function () {
+        var toolStore = this.flux.store("tool"),
+            superselectVector = toolStore.getToolByID("superselectVector");
+
+        return this.transfer(tools.select, superselectVector)
+            .then(function () {
+                return descriptor.playObject(vectorMaskLib.editVectorMask());
+            }).then(function () {
+                return descriptor.playObject(vectorMaskLib.activateVectorMaskEditing());
```

Comment:
Also, can you leave a note above each playObject call that summarizes what each one does? They have confusingly similar names...